src/pages/admin-settings.tsx

```diff
@@ -0,0 +1,319 @@
+        <div className='bg-white p-4 rounded-lg'>
+          {modalContent}
+          </div>
+          </div>
+          )}
+      <div className='flex justify-between p-4'>
+        <h2>Admin Dashboard</h2>
+        <h2>Welcome Back, Nhi</h2>
+      </div>
+      <div className='p-4'>
+  <div className='flex justify-between mb-4'>
+    <div>
+      <button
+        className={`py-2 px-4 rounded ${
+          currentTable === 1 ? 'bg-purple-600 text-white' : 'bg-gray-200 text-purple-800'
+        }`}
+        onClick={() => setCurrentTable(1)}
+      >
+        Users
+      </button>
+      <button
+        className={`py-2 px-4 rounded ml-2 ${
+          currentTable === 3 ? 'bg-purple-600 text-white' : 'bg-gray-200 text-purple-800'
+        }`}
+        onClick={() => setCurrentTable(3)}
+      >
+        Task Queues
+      </button>
+    </div>
+    {currentTable === 3 && (
+      <button
+        className='bg-purple-600 text-white py-2 px-4 rounded mb-4'
+        onClick={() => {
+          setModalContent(<AdminEditTask setShowModal={setShowModal} />);
+          setShowModal(true);
+        }}
+      >
+        Create New Task Queue
+      </button>
+    )}
+  </div>
+        {currentTable === 3 && (
+             <div className='flex justify-end mb-4'>
+             <div className='relative'>
+               <button
+                 className='border border-gray-400 rounded px-4 mr-2 relative'
+                 onClick={() => setShowSortOptions(!showSortOptions)}
+                 style={{ height: '30px', paddingTop: '4px', paddingBottom: '4px' }}
+               >
+                  ▼Sort
+                 <div
+                   className={`absolute top-full left-0 bg-white border border-gray-400 rounded-lg p-2 ${
+                     showSortOptions ? '' : 'hidden'
+                   }`}
+                 >
+                   <button className='w-full text-left py-1 px-2 hover:bg-gray-200'>
+                     Sort by Task Name
+                   </button>
+                   <button className='w-full text-left py-1 px-2 hover:bg-gray-200'>
+                     Sort by SID
+                   </button>
+                 </div>
+               </button>
+             </div>
+             <input
+               type='text'
+               placeholder='Type to search'
+               className='border border-gray-400 rounded w-full p-2 mr-2'
+               style={{ height: '30px' }}
+             />
+             <span style={{ fontSize: '24px' }}>🔍</span>
+           </div>                 
+        )}
+
+        {currentTable === 3 ? (
+          <div className='w-full m-auto p-4 border rounded-lg bg-white overflow-y-auto'>
+            <div className='my-3 p-2 grid grid-cols-2 items-center justify-between cursor-pointer'>
+              <span>Task Name/SID</span>
+              <span className='text-right'>Assigned Users</span>
```

Comment:
Would it be possible to add more between "Assigned Users" and the 3 dots?

Reply:
![Task Queues(2)](https://github.com/ConnieML/Connie-RTC/assets/47403443/f021dae8-665b-4e9e-b9d8-04f34c40dd21)